src/app/api/chapter-request/route.schema.ts

```diff
@@ -0,0 +1,84 @@
+import { z } from "zod";
+
+/*
+  firstName            String
+  lastName             String
+  universityEmail      String    @unique
+  phoneNumber          String
+  university           String
+  universityAddress    String
+  leadershipExperience String
+  motivation           String
+  availabilities       String
+  questions            String
+*/
+export const ChapterRequest = z.object({
+  firstName: z.string(),
+  lastName: z.string(),
+  universityEmail: z.string().email("This is not a valid email"),
+  phoneNumber: z.string(),
```

Comment:
We can enforce that phone numbers are valid. For now, I think it's safe to assume that the numbers are in United States - i.e 10 digits number